ocean_lib/data_provider/data_service_provider.py

```diff
@@ -221,6 +221,7 @@ def start_compute_job(
         output: dict = None,
         input_datasets: list = None,
         job_id: str = None,
+        raw_response: bool = False,
```

Comment:
I think it's better to not add unnecessary argument. We can just always return the `raw_response`. then the caller can take the job_id from that. Of course we have to update all existing calls to this method.